frontend/animal/AnimalGroupTable/AnimalGroupTable.js

```diff
@@ -0,0 +1,181 @@
+import _ from "lodash";
+import React, {Component} from "react";
+import PropTypes from "prop-types";
+import {inject, observer} from "mobx-react";
+
+import h from "shared/utils/helpers";
+
+const nRow = (endpoint, index) => {
+        return (
+            <tr key={`n-${index}`} className="table-secondary">
+                <td>N</td>
+                <td>-</td>
+                <td>-</td>
+                {endpoint.data.groups.map((d, i) => (
+                    <td key={i}>{d.n || "-"}</td>
+                ))}
+            </tr>
+        );
+    },
+    renderEndpointRow = (store, endpoint) => {
+        const {footnotes} = store,
+            {id, url, name, groups, observation_time_text, organ, data_type} = endpoint.data;
+        let dr_control, html, fn, response;
+        return (
+            <tr key={`e-${id}`} className="previewModalParent">
+                <td>
+                    <i
+                        className="float-right fa fa-fixed fa-eye previewModalIcon"
+                        onClick={() => endpoint.displayAsModal({complete: true})}></i>
+                    <a href={url}>{name}</a>
+                </td>
+                <td>{organ}</td>
```

Comment:
Right now if there's no `organ` it just leaves the cell empty, which is stylistically different than `nRow` which fills in empty cells with `-`. Not a big deal, but thought I'd point it out anyway.
![image](https://user-images.githubusercontent.com/46504563/134023441-9c6ea5d5-e0ca-436f-beaa-ce3fd4595339.png)

Reply:
ah, good find. i restored the "-" functionality.